STEERING-COMMITTEE.md

```diff
@@ -1,27 +1,108 @@
+
+1. Establishing rules of governance for the Istio project, including creation
+and ratification of bylaws for the Steering Committee and Technical Oversight
+Committee.
+1. Establishing and enforcing principles to guide the Istio project.
+1. Fostering an environment for a healthy and happy community of developers,
+contributors, and users.
+1. Defining, evolving, and defending a
+[Code of Conduct](CONTRIBUTING.md#code-of-conduct).
+1. Advising the trademark owner on issues relating to the Istio trademark and
+logo.
+1. Setting marketing and advocacy direction for the project; establishing a
+publishing schedule and vetting content, encouraging and assisting in project
+community members, fostering an ecosystem of vendors, creating content for
+conferences, etc.
+1. Controlling and delegating access to, and establishing processes regarding,
+project resources/assets, including but not limited to artifact repositories,
+build and test infrastructure, web sites and their domains, blogs, and social
+media accounts.
+1. Providing neutral mediation as appropriate to try to resolve non-technical
+disputes that arise as part of the project.
+
+## Membership and voting
+1.  The Steering Committee is structured to allow companies who are most
+invested in the success of the Istio project to participate in business and
+non-technical decision-making.
+1.  All members should abide by the project Code of Conduct.
+1.  There are two types of seats on the Steering Committee: Contributor Seats
+and Community Seats.
+1.  Both Contributor and Community Seats will be appointed beginning in
+July 2020. The appointments for the Contribution and Community seat types will
+expire on staggered dates. After the initial term for both seats, all seats
+will have an annual term.
+1.  Contributor Seat terms expire on January 1 and Community Seat terms expire
+on July 1. If necessary, a company holding a Contributor Seat may change the
+appointed individual at any time during the term.
+1.  No Contributing Company can have more than 5 seats in total on the Steering
+Committee.
+    1.  Subsidiaries, an entity in which another company owns more than 50%
+    of the voting or membership interests; and related companies, companies
+    which are controlled by a common third party entity; and affiliate
+    companies, will be treated together as a single entity, referred to as a
+    Contributing Company.
+1.  There shall be nine **Contributor Seats**. Their allocation is determined
+by the approximate effort and expenditure on the Istio project, as approximated
+by the number of merged pull requests on GitHub over the one year period prior
+to the Contributor Seat assignments:
+    1.  The top three contributing companies to Istio are eligible for
+    Contributor Seats, proportional to their contribution.
+        1.  Each company is allocated one seat;
+        1.  The remaining six seats are allocated based on percentage project
```

Comment:
I do not agree with the proportional seating allocation for steering. I’m open to the idea that contributors meeting the 5% qualification get a set number of seats (1 or 2) but no additional seats should be allocated based on contribution. A goal of the steering is to have fare and neutral representation.

Reply:
What about community-voted and contribution-based seats is not a fair make up that represents the community?

Reply:
Fair and neutral are not always compatible.  If the allocation is done based on meritocratic principles (percentage contributions), that could be called 'fair' but not 'neutral' when the contributions are not balanced.

In such a case though, there would be a path to neutral representation which would be increasing contributions to balance them out.   And it has the nice property that giving contributors incentives to increase their contributions is good for the project overall.

Reply:
This needs a discussion and consensus in the steering committee and approval by the current steering committee members.

Reply:
Since contributor seats would be a new addition, it would be good to hear from folks not currently on Steering but are active contributors and potentially could be!

Reply:
My main concern was not with contributor seats. My questioning is the need to distribute the 6 remaining community seats proportionally based on contributions

Reply:
There are 4 community seats (2 elected by community and 2 appointed by SC) and 9 contributor seats. The contributor seats are held by contributing _companies_. This thread was about the 6 remaining contributor seats after the top 3 companies are assigned a seat.

Reply:
Based on discussion at recent Steering committee meetings, we have consensus for 6 Contribution Seats, and 4 Community Seats. 

* Contribution seats are allocated based on merged PRs in the first instance, and that will be reviewed every year.
* Community seats are now all voted by the project members.

Reply:
@dcberg and @rvennam are you OK with me resolving this now?